src/main/java/seedu/duke/TaskList.java

```diff
@@ -0,0 +1,143 @@
+package seedu.duke;
+
+import seedu.duke.tasks.*;
```

Comment:
Should explicitly importing all classes needed better as required by given Java coding standard?

Reply:
Yup will change this! I used * because I thought that if there's more than 3 imports from the same package we can just replace with * but i guess not hahaha